_data/sites.json

```diff
@@ -2693,6 +2693,15 @@
         ]
     },
 
+    {
+        "name": "exercism",
+        "url": "https://exercism.io/my/settings/delete_account",
```

Comment:
This page returns 404 if you're not logged in. Is this hard to find? I'd rather have a link to settings, maybe, because it redirects if not logged in